examples/resources/js/components/ExampleComponent.vue

```diff
@@ -17,11 +17,11 @@
 <script>
     export default {
         async created() {
-            const { data: permissions } = await axios.get('/api/permissions');
-            const { data: roles } = await axios.get('/api/roles');
+            const { data: permissions } = axios.get('/api/permissions');
```

Comment:
@garretmh I think this code would fail because you're returning a `Promise`, not an object.

Reply:
Whoops! I should have looked at it before committing.

Reply:
@garretmh When using `await` in each promise no longer becomes parallel, ideally:

```javascript
await Promise.all([
  new Promise(async (resolve) => {
    const { data: permissions } = await axios.get('/api/permissions');
    this.$laravel.setPermissions(permissions);
    resolve();
  }),
  new Promise(async (resolve) => {
    const { data: roles } = await axios.get('/api/roles');
    this.$laravel.setRoles(roles);
    resolve();
  }),
]);
```

https://developer.mozilla.org/es/docs/Web/JavaScript/Referencia/Objetos_globales/Promise/all